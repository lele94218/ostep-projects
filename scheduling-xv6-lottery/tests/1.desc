simple test of lottery scheduling
