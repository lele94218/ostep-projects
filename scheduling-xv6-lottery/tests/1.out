XV6_TEST_OUTPUT:100
XV6_TEST_OUTPUT:10
XV6_TEST_OUTPUT:1
